spyder/plugins/ipythonconsole/widgets/help.py

```diff
@@ -139,10 +147,9 @@ def _handle_inspect_reply(self, rep):
             if content.get('status') == 'ok' and content.get('found', False):
                 signature = self.get_signature(content)
                 documentation = self.get_documentation(content)
-                if signature:
-                    self._control.show_calltip(
-                        signature,
-                        documentation=documentation,
-                        language=self.language_name,
-                        max_lines=5
-                    )
+                self._control.show_calltip(
+                    signature,
+                    documentation=documentation,
+                    language=self.language_name,
+                    max_lines=5
```

Comment:
Let's change this to `7` because several docstrings are cutoff at the `Parameters` line.